edenia/antelope-tools
# Feat/limit evm history

### Limit the period of time EVM data is retained

### What does this PR do?

- Avoid saving data from blocks older than one year.
- Fix the synchronization of the last blocks using the last block in the database, some blocks are ignore when using the endpoint directly.
- Calculate maximum statistics such as total incoming/outgoing tokens, total transactions and TPS all time high.

### Steps to test

1. Run the project locally with EVM configuration
2. Check that there are no old data and services are syncing correctly
